src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -247,10 +247,47 @@
                                 </tr>
 
                                 {% if collector.supportsRoleHierarchy %}
+                                <script>
```

Comment:
perhaps check if the data is not null/not empty before loading all this?

Reply:
That's a good idea indeed.